frontend/src/containers/SignUpContainer/SignUpForm.js

```diff
@@ -0,0 +1,49 @@
+import React, { useCallback, useEffect } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { withRouter } from 'react-router-dom';
+import SignUp from '../../components/SignUp/SignUp';
+import { register, changeField, initalizeForm } from '../../modules/auth/auth';
+
+function SignUpForm({ history }) {
+  const dispatch = useDispatch();
+  const form = useSelector(({ auth }) => ({
```

Comment:
To fix `form.form` to `form`, change `const form` into `const { form }`

Please reference this link: https://wesbos.com/destructuring-objects/